scripts/generate_virtual_player.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+import random
+import os
+
+from player import VirtualPlayer
+from game_session import GameSession
+from models import RescorlaWagner, Qlearning
+
+def prepare_parameters(number_of_trials, num_of_params):
+    parameters = np.zeros((number_of_trials, num_of_params))
+    if num_of_params == 3:
+        for params in range(parameters.shape[0]):
+            parameters[params][0] = random.random()
+            parameters[params][1] = random.uniform(-10,10)
+            parameters[params][2] = random.uniform(-10,10)
+    else:
+        for params in range(parameters.shape[0]):
+            parameters[params][0] = random.random()
+            parameters[params][1] = random.uniform(-10,10)
+
+    return parameters
+
+
+def _create_file_path(filename):
+    parent_directory = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+    return os.path.join(parent_directory, "data", filename)
+
+
+if __name__=='__main__':
+    game = GameSession()
+    params = prepare_parameters(10,3)
+    attempt = params[0,:]
```

Comment:
attempt is also a misleading name here, why would you create 10 sets of parameters and then use only one? this regards also comment about lacking function for generating multiple csv